ckanext/unaids/react/components/BulkFileUploaderComponent/src/ModalBody.js

```diff
@@ -167,13 +167,11 @@ export default function ModalBody({
                                         : fileUploadActionSelector(file, index))
                                 }
                             </td>
-                            <td width={20}>
-                                {!file.progress &&
-                                    <i
-                                        className="fa fa-close text-danger remove-file-btn"
-                                        onClick={() => removeFileFromPendingFiles(index)}
-                                    ></i>
-                                }
+                            <td width={30}>
+                                <i
+                                    className={`fa fa-close remove-file-btn-${uploadInProgress ? 'disabled' : 'active text-danger'} `}
```

Comment:
The modal x is hidden, but these are disabled.  Is there a reason why you havn't just hidden these as well?

Reply:
Yeah that's a bit better, i'll go ahead with that